code/game/objects/items/weapons/implant/implants/carrion/infection_spider.dm

```diff
@@ -19,6 +19,10 @@
 	if(is_neotheology_disciple(wearer))
 		to_chat(owner_mob, SPAN_WARNING("[wearer]'s cruciform prevents activation"))
 		return
+	var/obj/item/organ/external/affected = wearer.organs_by_name[pick(list(BP_HEAD, BP_CHEST))]
```

Comment:
Pick picks one random item from the list, why use it here?

Reply:
removed the pick, made it even dumber, love you